recipes/lcms2/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: lcms2
+  version: 2.7
+
+source:
+  fn: lcms2-2.7.tar.gz
+  url: https://sourceforge.net/projects/lcms/files/lcms/2.7/lcms2-2.7.tar.gz
+  sha256: 4524234ae7de185e6b6da5d31d6875085b2198bc63b1211f7dde6e2d197d6a53
+
+build:
+  number: 0
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - cmake         # [win]
+    - jpeg 8d
+    - libtiff 4.0.6
```

Comment:
Unless you really need to be this specif `libtiff 4.0.*` is recommend. Same for below `zlib 1.2.*`